src/components/ttt-tile/ttt-tile.scss

```diff
@@ -104,3 +104,7 @@
     height: 1rem;
     vertical-align: text-bottom;
 }
+
+h3 {
+    font-size: 1rem;
+}
```

Comment:
Won't this apply to all `h3` elements on the page, not only those in the `ttt-tile` component? It should probably be added to the `.ttt-tile-title` class instead.

Reply:
Hey, @mxmou, thanks for catching that. I made the requested change.